lumberdev/atelier
# Footer nav menu component in app

### WHAT IT DOES:

- Created a footer nav menu component for customizing the footer links for campaign pages

### WHAT TO TEST:

- [ ] In Atelier app, go to **Settings** tab, and scroll to **Store Footer** section
- [ ] Try playing around with the _Footer Link Navigation_ field, specifically, try these things:
 - Add a menu item
 - Edit / Delete fields
 - Drag & Drop the link menus 
 - Reload the page to check if the changes persist
- [ ] Go to any campaign page in website, and check if the footer navigation has been reflected. 

### PREVIEW:

- Empty state

![image](https://github.com/lumberdev/atelier/assets/43429460/dc67a096-9ec4-447f-ac93-68ef724d2164)

- Adding field

![image](https://github.com/lumberdev/atelier/assets/43429460/893ebb46-26ef-4f5e-924b-32438fd7d579)

- Field populated

![image](https://github.com/lumberdev/atelier/assets/43429460/070608e3-d9ae-413d-b196-f9255600e760)

- Editing field

![image](https://github.com/lumberdev/atelier/assets/43429460/19c9a256-2f0d-4517-8560-f50cdda5f24e)

- Moving menus

![image](https://github.com/lumberdev/atelier/assets/43429460/2f4a8bea-0ad0-4d9b-83e2-b003f4c18fb6)
